### PR TITLE
[import wizard] update flow and layout of pages according to mocks

### PR DESCRIPTION
**Pull request by chejennifer**
- added new download page
- move upload file input to the same page as template selection
- move some of the state to Page component

screen-recording-_92_

Notes:
- some files need to be renamed because their purpose has changed slightly, but will do this in a follow up PR

### CODE REVIEW

File: static/js/import_wizard2/components/download_page.tsx [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+  TMCF: "tmcfFile",
+  METADATA: "translationMetadata",
+  CLEANED_CSV: "cleanedCsv",
+  USER_CSV: "userCsv",
+  ALL: "allFiles",
+};
+
+// Map of file type key to the label to use to describe the file.
+const FILE_TYPE_LABEL = {
+  [FILE_KEYS.TMCF]: "Generated tMCF file",
```
Comment (shifucun) [inline static/js/import_wizard2/components/download_page.tsx:56]: nit: s/tMCF/TMCF for consistency

File: static/js/shared/util.ts [hunk 2]
```diff
@@ -56,29 +56,38 @@ export function randDomId(): string {
...
+
 /**
  * Saves csv to filename.
  * @param {filename} string
  * @param {contents} string
  * @return void
  */
 export function saveToFile(filename: string, contents: string): void {
-  let mimeType = "text/plan";
+  let mimeType = "text/plain";
```
Comment (shifucun) [inline static/js/shared/util.ts:83]: wonder what issue this has now?

Comment (chejennifer) [reply static/js/shared/util.ts:83]: surprisingly, no issues .. but "text/plan" is not a mimeType

File: static/css/import_wizard2.scss [hunk 3]
```diff
@@ -364,4 +351,69 @@ tr th:last-child {
     background-color: #ccc;
     border-radius: 1rem;
   }
+}
+
+.file-upload-section {
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/css/import_wizard2.scss:358]: Maybe can do this in a follow up PR? added a TODO

File: static/js/import_wizard2/components/download_page.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/import_wizard2/components/download_page.tsx]: done

**Review by shifucun — APPROVED** [review]

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**